pkg/factory/risingwave_object_factory.go

```diff
@@ -550,8 +550,39 @@ func (f *RisingWaveObjectFactory) envsForMetaArgs() []corev1.EnvVar {
 	return envVars
 }
 
+func (f *RisingWaveObjectFactory) envsForTLS() []corev1.EnvVar {
+	tls := f.risingwave.Spec.TLS
+	if tls != nil && tls.SecretName != "" {
+		return []corev1.EnvVar{
+			{
+				Name: envs.RWSslKey,
+				ValueFrom: &corev1.EnvVarSource{
+					SecretKeyRef: &corev1.SecretKeySelector{
+						LocalObjectReference: corev1.LocalObjectReference{
+							Name: tls.SecretName,
+						},
+						Key: corev1.TLSPrivateKeyKey,
+					},
+				},
+			},
+			{
+				Name: envs.RWSslCert,
+				ValueFrom: &corev1.EnvVarSource{
+					SecretKeyRef: &corev1.SecretKeySelector{
+						LocalObjectReference: corev1.LocalObjectReference{
+							Name: tls.SecretName,
+						},
+						Key: corev1.TLSCertKey,
+					},
+				},
+			},
+		}
+	}
+	return nil
+}
+
 func (f *RisingWaveObjectFactory) envsForFrontendArgs() []corev1.EnvVar {
-	return []corev1.EnvVar{
+	envVars := []corev1.EnvVar{
```

Comment:
Shall we update test cases for this change?

Reply:
Sure, thanks for reminding! Just added.